soc/xtensa/espressif_esp32/esp32s2/default.ld

```diff
@@ -133,7 +135,7 @@ SECTIONS
    * Adding .rodata as first section helps to reduce size of generated binary by
    * few kBs.
    */
-  SECTION_PROLOGUE(_RODATA_SECTION_NAME,,ALIGN(4))
+  SECTION_PROLOGUE(_RODATA_SECTION_NAME,,)
```

Comment:
What exactly is the meaning of removing all those alignments, and what is the impact on the resulting binary regarding the mentioned issue?